liberty-archetype-ear/src/main/resources/archetype-resources/__rootArtifactId__-ear/src/main/liberty/config/server.xml

```diff
@@ -6,29 +6,10 @@
         <feature>javaee-7.0</feature>
     </featureManager>
 
-    <!-- This template enables security. To get the full use of all the capabilities, a keystore and user registry are required. -->
-    
-    <!-- For the keystore, default keys are generated and stored in a keystore. To provide the keystore password, generate an 
-         encoded password using bin/securityUtility encode and add it below in the password attribute of the keyStore element. 
-         Then uncomment the keyStore element. -->
-    <!--
-    <keyStore password=""/> 
-    -->
-    
-    <!--For a user registry configuration, configure your user registry. For example, configure a basic user registry using the
-        basicRegistry element. Specify your own user name below in the name attribute of the user element. For the password, 
-        generate an encoded password using bin/securityUtility encode and add it in the password attribute of the user element. 
-        Then uncomment the user element. -->
-    <basicRegistry id="basic" realm="BasicRealm"> 
-        <!-- <user name="yourUserName" password="" />  --> 
-    </basicRegistry>
-    
```

Comment:
I thought the javaee-7 feature required some security configuration.  Do you not get errors with this configuration?